src/argilla/client/feedback/dataset/remote/base.py

```diff
@@ -323,4 +331,13 @@ def push_to_argilla(
             UserWarning,
             stacklevel=2,
         )
-        pass
+        raise NotImplementedError(
+            "`push_to_argilla` doesn't work for neither `RemoteFeedbackDataset` nor `FilteredRemoteFeedbackDataset`.",
+            "First call `pull_from_argilla` to get a local instance of the dataset, and then call `push_to_argilla(*args, **kwargs)`",
+        )
```

Comment:
I'd say we don't need to `raise NotImplementedError`, I think the exception is far more than enough and it doesn't break any existing snippet

Reply:
```suggestion
```